pkg/migrations/sql/idempotent/base.sql

```diff
@@ -2754,7 +2762,7 @@ BEGIN
         WHERE h.schema_name = 'SCHEMA_DATA' and h.table_name = ht_table;
 
         --alter job schedule is not currently concurrency-safe (timescaledb issue #2165)
-        PERFORM pg_advisory_xact_lock(ADVISORY_LOCK_PREFIX_JOB, bgw_job_id);
+        PERFORM pg_advisory_xact_lock(12377, bgw_job_id);
```

Comment:
Why did we not create a function for this one? Is it because this is the only instance of the number?

Reply:
That was the idea. I can also pull this into a function if you would prefer.

Reply:
I've pulled this into a function